draft-ietf-scitt-architecture.md

```diff
@@ -1293,3 +1294,60 @@ transparent-statement:sha-256:base64url:5i6UeRzg1...qnGmr1o
 data:application/cose;base64,SGVsb...xkIQ==
 ~~~
 {: #example-transparent-statement-data-url align="left" title="Example Transparent Statement Data URL"}
+
+# Signing Statements Remotely
+
+Statements, such as digital artifacts or structured data regarding artifacts, can be too large or too sensitive to be send to a remote Transparency Services over the Internet.
+In these cases a statement can also be hash, which becomes the payload included in COSE to-be-signed bytes.
+A Signed Statement (cose-sign1) MUST be produced from the to-be-signed bytes according to {{Section 4.4 of RFC9052}}.
+
+~~~aasvg
+
+ .----+-----.
+|  Artifact  |
+ '----------'
+    |   |
+    v   |               .
+ .--+-------.          / \
+| Statement  +--+     /   \
+ '----------'   |    /     \     .----------.
+        |       +-->+  OR   +-->+  Payload   |
+        v       |    \     /     '---+------'
+ .------+---.   |     \   /          |
+|    Hash    +--+      \ /           |
+ '----+-----'           '            |
+                                     |
+
+           ...  Producer Network ...
+
+                      ...
+
+           ...   Issuer Network ...
+
+                                     |
++-----------+                        |
+| Identity  |     (iss, x5t)         |
+| Document  +--------------------+   |
++-----+-----+                    |   |
+      ^                          |   |
+      |                          |   |
+ .----+-------.                  |   |
+| Private Key  |                 |   |
+ '----+-------'                  v   |
+      |                     .----+-----.
+      |                    |   Header   |
+      |                     '----+-----'
+      |                          |   |
+      v                          v   v
+    .-+-----------.     .--------+---+---.
+   /             /     /                  \
+  /    Sign     +<----+ To Be Signed Bytes |
+ /             /       \                  /
+'-------------'         '----------------'
```

Comment:
```suggestion
'-----+-------'         '----------------'
```

Reply:
Minor fix to differentiate lines "behind", vs. from. 
In this case, I believe @OR13 is showing the COSE Sign 1 Document comes from the Sign process.

Reply:
The lines are a correct as they are.